extensions/wikia/InstantGlobals/InstantGlobalsHooks.class.php

```diff
@@ -3,8 +3,9 @@
 class InstantGlobalsHooks {
 
 	public static function onWikiaSkinTopShortTTLModules( Array &$modules, $skin ) {
-		if ( F::app()->checkSkin( ['venus', 'oasis'], $skin ) ) {
+		if ( F::app()->checkSkin( ['venus', 'oasis', 'wikiamobile'], $skin ) ) {
 			$modules[] = 'wikia.ext.instantGlobals';
+			$modules[] = 'wikia.ext.instantGlobalsOverride';
```

Comment:
Decided not to use array_push with 2 items here in case $modules array is not defined. What do you think?